libs/cdk/src/lib/zone-less/utils.ts

```diff
@@ -1,39 +1,27 @@
-import { ɵglobal as globalThis } from '@angular/core';
-
-/**
- *
- * utils.ts
- *
- * To use zone un-patched versions of the following Window methods:
- *
- * 1. Add import of the function to your component, service, etc.
- *
- * import { setTimeout } from '@cu/perf-utils';
- *
- * 2. Use the it the same way as the original ("zone patched") version
- *
- * triggerAnimation(direction: string): void {
- *   this.animate = direction;
- *   setTimeout(() => (this.animate = 'reset'), 185);
- * }
- *
- */
-
-type obj = Record<string | number | symbol, any>;
+import { ɵglobal } from '@angular/core';
 
 /**
  * @description
  *
  * This function returns the zone un-patched API for the a specific Browser API.
- * If no element is passed the window is used instead
+ * If no target is passed the window is used instead
  *
- * @param name {string} - The name of the API to check.
- * @param elem {any} - The elem to get un-patched API from.
+ * @param name - The name of the API to check.
+ * @param target - The target to get un-patched API from.
  * @return {Function} - The zone un-patched API in question.
  *
  */
-export function getZoneUnPatchedApi<T = Function>(name: string, elem?: obj): T {
-  elem = elem || (globalThis as obj);
-  const isPatched = elem['__zone_symbol__' + name] !== undefined;
-  return isPatched ? elem['__zone_symbol__' + name] : elem[name];
+export function getZoneUnPatchedApi<
+  N extends keyof (Window & typeof globalThis)
+>(name: N): (Window & typeof globalThis)[N];
+export function getZoneUnPatchedApi<T extends object, N extends keyof T>(
+  name: N,
+  target: T
+): T[N];
+export function getZoneUnPatchedApi<T extends object, N extends keyof T>(
+  name: N,
+  target?: T
+): T[N] {
+  target = target || ɵglobal;
```

Comment:
What about defining a default target as following :

```suggestion
  target: T = ɵglobal
): T[N] {
```

Reply:
Thanks, updated!